src/plugins/analysis/cve_lookup/test/test_setup_repository.py

```diff
@@ -1047,7 +1047,7 @@
 ]
 
 
-@pytest.fixture(scope='session', autouse=True)
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Not your code so not your fault but shouldn't this function also use monkeypatch for `sr.QUERIES.update(TEST_QUERIES)`?

Reply:
> shouldn't this function also use monkeypatch?

It should but we won't change this in this pr.